oe_theme.theme

```diff
@@ -819,7 +819,7 @@ function oe_theme_preprocess_paragraph__oe_list_item(array &$variables): void {
   $cacheability = CacheableMetadata::createFromRenderArray($variables);
 
   // Extract the image if present.
-  if (!$paragraph->get('field_oe_image')->isEmpty()) {
+  if (!$paragraph->get('field_oe_image')->isEmpty() && $variables['variant'] !== 'default') {
```

Comment:
I didn't get why the image is not allowed to default variant. In templates/patterns/list_item/pattern-list-item.html.twig image is used for the picture variable.